_includes/cloud/note-static-content-deploy-config.md

```diff
@@ -0,0 +1,3 @@
+
+  {:.bs-callout-info}
+  The `app/etc/config.php` file includes a _scopes_ setting that defines how static files deploy during the build phase. By default, the scope is [quick]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick). Static file deployment takes a long time to complete, so doing it during the build phase reduces deployment and site downtime. On the Cloud platform, you can customize settings in the YAML configuration files to minimize site downtime when deploying to Staging and Production environments. See [Optimize cloud deployment]({{ page.baseurl }}/cloud/deploy/optimize-cloud-deployment.html).
```

Comment:
config.php file does not define the strategy for SCD.
This file contains only locales for which static files should be generated and section `scopes` where information about stores is localed

Reply:
Removed this note because info is explained elsewhere, and it's not relevant for the context of the topic.